swap/src/main.rs

```diff
@@ -25,16 +25,18 @@ use tracing::{info, log::LevelFilter};
 use uuid::Uuid;
 
 pub mod bitcoin;
-mod cli;
 pub mod config;
 pub mod database;
-mod fs;
 pub mod monero;
 pub mod network;
 pub mod protocol;
 pub mod seed;
 pub mod trace;
 
+mod cli;
+mod fs;
+mod serde_peer_id;
```

Comment:
We should consider upstreaming a `serde` feature to `libp2p`.

Reply:
yes :)